helm/ffc-pay-statement-generator/values.yaml

```diff
@@ -39,6 +39,8 @@ container:
   azureStorageCreateContainers: false
   storageContainer: statements
   storageFolder: outbound
+  statementVersion: v1
+  statementReceiverEndpoint: http://ffc-pay-statement-receiver.ffc-pay
```

Comment:
This will need overwriting via a Value from App Config once we have DNS and App Gateway setup.

I can't find a ticket that covers that so can you make sure that we have one that includes:
- Setup App Gateway
- Setup DNS
- Update App Config with ingress values for `ingress.server` and `ingress.endpoint` for `ffc-pay-statement-receiver`
- Update App Config with corresponding values in this repository